Lib/test/test_os.py

```diff
@@ -1133,12 +1133,12 @@ def test_key_type(self):
 
         with self.assertRaises(KeyError) as cm:
             os.environ[missing]
-        self.assertIs(cm.exception.args[0], missing)
+        self.assertIn(missing, cm.exception.args[0])
```

Comment:
Did you intend to make this change?  If so, why?

Reply:
Now that the error message has more than just the key repr in it, I use `assertIn` to ensure that the error message `__contains__` the repr of `missing`.

Reply:
(see updated code, which includes `repr`)